tests/test_complex.py

```diff
@@ -24,6 +27,30 @@
 EXP = gen_fexp(BitVector.get_family())
 Add32 = gen_Add32(BitVector.get_family())
 Sub32 = gen_Sub32(BitVector.get_family())
+RoundToZeroBounded = gen_round_to_zero_bounded(BitVector.get_family())
+RoundToZero = gen_round_to_zero(BitVector.get_family())
+
+_BOUND = 2**15-2**6-1
+_bounded_args = []
+while len(_bounded_args) < NTESTS:
+    arg = BFloat16.random()
+    while arg > _BOUND or arg < -_BOUND:
+        arg = BFloat16.random()
+    _bounded_args.append(arg)
+
+@pytest.mark.parametrize("arg", _bounded_args)
+def test_round_bounded(arg):
+    round_to_zero_bounded = RoundToZeroBounded()
+    r2z = round_to_zero_bounded(arg.reinterpret_as_bv())
+    gold = BFloat16(gmpy2.rint_trunc(arg._value))
```

Comment:
Could this be a method on the FPVector class instead?

Reply:
Yes, but that is a hwtypes issue not a lassen issue.  We currently use the types rounding mode when performing fp_round_to_integral but it could be a argument instead.